examples/seq2seq/README.md

```diff
@@ -17,23 +18,33 @@ export XSUM_DIR=${PWD}/xsum
 this should make a directory called `xsum/` with files like `test.source`.
 To use your own data, copy that files format. Each article to be summarized is on its own line.
 
-CNN/DailyMail data
+#### CNN/DailyMail
 ```bash
 cd examples/seq2seq
-wget https://s3.amazonaws.com/datasets.huggingface.co/summarization/cnn_dm.tgz
-tar -xzvf cnn_dm.tgz
+wget https://s3.amazonaws.com/datasets.huggingface.co/summarization/cnn_cln.tgz
+tar -xzvf cnn_cln.tgz  # empty lines removed
```

Comment:
Does `cnn_cln` include DailyMail? This filename is a little confusing to me.

Reply:
yeah I can fix it!